bndtools.m2e/src/bndtools/m2e/MavenImplicitProjectRepository.java

```diff
@@ -69,6 +50,12 @@ public MavenImplicitProjectRepository(IMavenProjectFacade projectFacade, File bn
 		this.bndrunFile = bndrunFile;
 		this.run = run;
 
+		fileSetRepository = ofNullable(run.getProperties()
```

Comment:
Since we now add the BndrunContainer object to the run's plugins, this now becomes

```java
fileSetRepository = ofNullable(run.getPlugin(BndrunContainer.class))
  .map(asFunction(BndrunContainer::getFileSetRepository))
  .orElse(null);
```

Reply:
totally.